### PR TITLE
Fixing #2427

### PR DESCRIPTION
**Pull request by JorjMcKie**
Change the wording to correctly reflect the picture.

### CODE REVIEW

**Comment by jamie-lemon** [comment]
Looking at the bug and the picture there isn't the final rotation 30 degrees clockwise ( not anti-clockwise ) ?

**Review by jamie-lemon — CHANGES_REQUESTED** [review]

File: docs/matrix.rst [hunk 1]
```diff
@@ -233,7 +233,7 @@ Rotating
 ---------
 .. |rot60| image:: images/img-rot-60.*
 
-Finally a rotation by 30 clockwise degrees (*prerotate(-30)*).
```
[the remark this answers is not in the document]
Comment (JorjMcKie) [reply docs/matrix.rst:236]: Hm - to be honest: that matrix is anti-clockwise was - a long time ago - simply copied from MuPDF description.

[the remark this answers is not in the document]
Comment (jamie-lemon) [reply docs/matrix.rst:236]: I'm confused - it looks like the image is rotated clockwise?

[the remark this answers is not in the document]
Comment (JorjMcKie) [reply docs/matrix.rst:236]: I have uploaded a revised version. This applies to text in the .rt file as well as replacing the images involved in visualizing matrix effects.

File: docs/matrix.rst [hunk 2]
```diff
@@ -35,7 +35,7 @@ Please note:
 :attr:`Matrix.d`                 zoom factor Y direction
 :attr:`Matrix.e`                 horizontal shift
 :attr:`Matrix.f`                 vertical shift
-:attr:`Matrix.is_rectilinear`    true if rect corners will remain rect corners
+:attr:`Matrix.is_rectilinear`     true if rect corners will remain rect corners
```
[the remark this answers is not in the document]
Comment (jamie-lemon) [reply docs/matrix.rst:38]: Loads more clear with those new images.

**Review by jamie-lemon — APPROVED** [review]
LGTM

**Pull request merged**